opal/mca/memory/linux/memory_linux_component.c

```diff
@@ -162,6 +187,59 @@ static int linux_register(void)
                                             OPAL_INFO_LVL_3,
                                             MCA_BASE_VAR_SCOPE_READONLY,
                                             &opal_memory_linux_disable);
+    if (0 > ret) {
+        return ret;
+    }
+
+#if MEMORY_LINUX_MALLOC_ALIGN_ENABLED
+    mca_memory_linux_component.use_memalign = 0;
+    ret = mca_base_component_var_register(&mca_memory_linux_component.super.memoryc_version,
+                                 "memalign",
+                                 "[64 | 32 | 0] - Enable (64bit or 32bit)/Disable(0) memory"
+                                 "alignment for all malloc calls (default: 0).",
+                                 MCA_BASE_VAR_TYPE_INT,
+                                 NULL,
+                                 0,
+                                 0,
+                                 OPAL_INFO_LVL_9,
```

Comment:
Should probably be level 5.

Also, if there are discrete values that this MCA param can accept, they should be itemized in an enumerator.